### PR TITLE
chore: cut v0.71.0

### PR DESCRIPTION
**Pull request by simonpasquier**
## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [X] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Verification

Please check the [Prometheus-Operator testing guidelines](../TESTING.md) for recommendations about automated tests.

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

**Review by ArthurSens — APPROVED** [review]
:shipit:

**Review by nicolastakashi — APPROVED** [review]

**Comment by nicolastakashi** [comment]
Let's goooo :shipit:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+* [FEATURE] Add support for OpenStack Service Discovery to the ScrapeConfig CRD. #6168
+* [FEATURE] Add support for all settings of the Kubernetes Service Discovery to the ScrapeConfig CRD. #6178
+* [FEATURE] Add support for proxy settings to the ScrapeConfig CRD. #6001
+* [FEATURE] Add the `enableHTTP2` field to remote-write configuration. #6192
+* [FEATURE] Add `.spec.maximumStartupDurationSeconds` to the Prometheus and PrometheusAgent CRDs. #6137
+* [FEATURE] Emit Kubernetes events when detecting invalid configurations. #6179
+* [ENHANCEMENT] Add `additionalLabels` field to topology spread constraints for Prometheus and PrometheusAgent CRD. #5967
+* [ENHANCEMENT] Add `.spec.clusterLabel` to the Alertmanager CRD. #6162
+* [ENHANCEMENT] Add `prometheus_operator_status_update_errors_total` and `prometheus_operator_status_update_operations_total` metrics. #6185
+
```
Comment (slashpai) [inline CHANGELOG.md:14]: @simonpasquier  Can we try to get in https://github.com/prometheus-operator/prometheus-operator/pull/6220 as well? :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
+## 0.71.0 / 2024-01-11
```
Comment (slashpai) [inline CHANGELOG.md:1]:
```suggestion
## 0.71.0 / 2024-01-12
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+
+* [CHANGE] Configure the configuration reloader sidecar with the same web configuration as Prometheus and Alertmanager. #6194
```
Comment (slashpai) [inline CHANGELOG.md:3]: We need to have https://github.com/prometheus-operator/prometheus-operator/pull/6222/files in CHANGE as well

Comment (slashpai) [reply CHANGELOG.md:3]: Just realised this change was added recently only so won't need 🙂

File: CHANGELOG.md (hunk 1 above)
Comment (simonpasquier) [reply CHANGELOG.md:14]: v0.71.0 is already a bit late and also packed. I suggest to postpone for the next one.

**Review by philipgough — APPROVED** [review]

**Pull request merged**